pkg/ui/react-app/src/thanos/pages/blocks/SourceView.tsx

```diff
@@ -2,6 +2,7 @@ import React, { FC } from 'react';
 import { Block, BlocksPool } from './block';
 import { BlockSpan } from './BlockSpan';
 import styles from './blocks.module.css';
+import { Popup } from 'semantic-ui-react';
```

Comment:
Use https://reactstrap.github.io/components/tooltips/ as we already use Reactstrap all over.